src/app/zap-templates/zcl/data-model/chip/occupancy-sensing-cluster.xml

```diff
@@ -50,69 +56,52 @@ limitations under the License.
+      </feature>
+      <feature bit="4" code="AIR" name="ActiveInfrared" summary="Supports sensing using Active InfraRed measurement (e.g. time-of-flight or transflective/reflective IR sensing)">
+        <optionalConform choice="a" more="true" min="1"/>
+      </feature>
+      <feature bit="5" code="RAD" name="Radar" summary="Supports sensing using radar waves (microwave)">
+        <optionalConform choice="a" more="true" min="1"/>
+      </feature>
+      <feature bit="6" code="RFS" name="RFSensing" summary="Supports sensing based on RF signal analysis">
+        <optionalConform choice="a" more="true" min="1"/>
+      </feature>
+      <feature bit="7" code="VIS" name="Vision" summary="Supports sensing based on analyzing images">
+        <optionalConform choice="a" more="true" min="1"/>
+      </feature>
     </features>
-
     <client tick="false" init="false">true</client>
     <server tick="false" tickFrequency="half" init="false">true</server>
-
     <globalAttribute side="either" code="0xFFFD" value="5"/>
-
-    <attribute side="server" code="0x0000" name="Occupancy" define="OCCUPANCY" type="OccupancyBitmap" min="0x00" max="0x01" reportable="true">
-      <mandatoryConform/>
-    </attribute>
-    <attribute side="server" code="0x0001" name="OccupancySensorType" define="OCCUPANCY_SENSOR_TYPE" type="OccupancySensorTypeEnum" min="0x00" max="0xFE">
-      <otherwiseConform>
-        <mandatoryConform/>
-        <deprecateConform/>
-      </otherwiseConform>
-    </attribute>
-    <attribute side="server" code="0x0002" name="OccupancySensorTypeBitmap" define="OCCUPANCY_SENSOR_TYPE_BITMAP" type="OccupancySensorTypeBitmap" min="0x00" max="0x07">
-      <otherwiseConform>
-        <mandatoryConform/>
-        <deprecateConform/>
-      </otherwiseConform>
-    </attribute>
-
-    <attribute side="server" code="0x0003" name="HoldTime" define="HOLD_TIME" type="int16u" writable="true" optional="true">
+    <attribute side="server" code="0x0000" name="Occupancy" define="OCCUPANCY" type="OccupancyBitmap" min="0x00" max="0x01">Occupancy</attribute>
+    <attribute side="server" code="0x0001" name="OccupancySensorType" define="OCCUPANCY_SENSOR_TYPE" type="OccupancySensorTypeEnum" min="0x00" max="0x03">OccupancySensorType</attribute>
+    <attribute side="server" code="0x0002" name="OccupancySensorTypeBitmap" define="OCCUPANCY_SENSOR_TYPE_BITMAP" type="OccupancySensorTypeBitmap" min="0x00" max="0x07">OccupancySensorTypeBitmap</attribute>
+    <attribute side="server" code="0x0003" name="HoldTime" define="HOLD_TIME" type="int16u" writable="true" optional="true" min="0" max="65535">
       <access op="write" privilege="manage"/>
+      <description>HoldTime</description>
       <optionalConform/>
     </attribute>
-
     <attribute side="server" code="0x0004" name="HoldTimeLimits" define="HOLD_TIME_LIMITS" type="HoldTimeLimitsStruct" optional="true">
+      <description>HoldTimeLimits</description>
```

Comment:
Why are name and description both present and set to the same thing?

Reply:
It seems that this was caused by a change in ZAP files where the "name" field inside the <attribute> element was added but was not considered during Alchemy generation, with the latest version the <description> element is removed if the "name" is configured.

Updated the files with the ones generated with the latest Alchemy version.